.dictionary.txt

```diff
@@ -1,3 +1,5 @@
 checkov
 Laven
+ltsc
+nanoserver
```

Comment:
I added these to the win32 dictionary.

Reply:
I tried removing those words and adding the win32 dictionary, but it looks like those words didn't make it into this release.  Hopefully soon!